cg/cli/add.py

```diff
@@ -58,7 +59,7 @@ def add():
     "--data-archive-location",
     "data_archive_location",
     help="Specifies where to store data for the customer.",
-    default="PDC",
```

Comment:
Im wondering if there should be a check that these locations are within a list of valid values. What if by accident you give "fastq" as a location? there will be spring files tagged as fast

Reply:
This is not part of this flow, I just added the constant and exchanged the hard coded string. So this does not affect the tagging, this is for when you add a customer. I don't see a way to add a restriction to that @diitaz93